test/unit/models/discord.test.js

```diff
@@ -0,0 +1,21 @@
+const chai = require("chai");
+const { expect } = chai;
+
+const cleanDb = require("../../utils/cleanDb");
+const discordQuery = require("../../../models/discord");
+const discordData = require("../../fixtures/discord/discord")();
+
+describe("discord", function () {
+  afterEach(async function () {
+    await cleanDb();
+  });
+
+  describe("addDiscordData", function () {
+    it("should add discord data to discord firestore collection", async function () {
+      const response = await discordQuery.addDiscordData(discordData[0]);
+
+      expect(response).to.be.an("object");
+      expect(response.message).to.equal("Added data successfully");
```

Comment:
let's check for the fields as well in the test.

Reply:
It doesn't contain anything else other than `message`
![image](https://user-images.githubusercontent.com/78433013/214642473-e734510c-dc7d-45bd-899f-b770c0cfb80c.png)

Reply:
ok